packages/ledger-plugin/go-ethereum/validator/src/core/bin/www.js

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2019-2020 Fujitsu Laboratories Ltd.
```

Comment:
Just a question because I really don't know: Doesn't the copyright belong to Hyperledger/Linux Foundation?

Reply:
In other Hyperledger projects, copyright notes include organizations which developers belong (e.g. IBM, Intel)
- e.g.) HL Fabric: https://github.com/hyperledger/fabric/blob/master/core/chaincode/active_transactions.go
- e.g.) HL Sawtooth: https://github.com/hyperledger/sawtooth-core/blob/master/rest_api/setup.py

Reply:
Ha! I had no idea, thanks @takeutak

Reply:
No.  Per Hyperledger policy any copyright needs to be for project contributors e.g. "Copyright contributors to Hyperledger Cactus"

https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy